torch/distributed/elastic/agent/server/local_elastic_agent.py

```diff
@@ -156,16 +156,16 @@ def _setup_local_watchdog(self, envs: Dict[int, Dict[str, str]]) -> None:
         if watchdog_enabled is not None and str(watchdog_enabled) == "1":
             if watchdog_file_path is None:
                 watchdog_file_path = "/tmp/watchdog_timer_" + str(uuid.uuid4())
-            log.info("Starting a FileTimerServer with %s ...", watchdog_file_path)
+            logger.info("Starting a FileTimerServer with %s ...", watchdog_file_path)
```

Comment:
should watchdog logging all debug level? cc @wconstab 
I think from a user prospective they don't know what's a watch dog until watchdog brings down processes.

Reply:
I changed the 3 logs in `local_elastic_agent.py` to `debug` but can wait to see what @wconstab might have to see. Have a look and see what you think.

Reply:
well tbh im not even that familiar with _this_ watchdog (the one in elastic, rather than the one in PGNccl).

I'd guess wanchao is right, but @kurman would be more informed.  no objection from me to move to debug here.

Reply:
We don't configure it by default, I think info level make sense.